test/functionalTest/cases/1887_hash_in_url_gives_strange_results/hash_in_url_gives_strange_results.test

```diff
@@ -20,6 +20,16 @@
 
 # VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
 
+#
+# NOTE: where before the hash sign were used, now we use a '<' sign.
```

Comment:
Maybe a bit excesive, but it doesn't hurt anyway... ;)

However, if we keep it, I'd suggest to include the SOF post where this is explained. I mean: https://stackoverflow.com/questions/46281537/why-modern-curl-versions-cut-off-the-anchor-part-of-the-url

Reply:
Fixed in 0768c23